exonum-java-binding-core/src/main/java/com/exonum/binding/proxy/ProxyDestructor.java

```diff
@@ -107,4 +108,14 @@ public void clean() {
   public Optional<Class<?>> resourceType() {
     return Optional.of(proxyClass);
   }
+
+  @Override
+  public String toString() {
```

Comment:
Please remove `nativeHandle` — it doesn't override toString + it is zeroed once closed;
and `cleanFunction` — it is usually a lambda, which `#toString` representation is not informative.

I'd keep `proxyClass` and `destroyed`, but consider if we shall make `NativeHandle` keep the original value of the handle to improve debugability + implement toString :thinking: